packages/driver/test/unit/cypress/stack_utils.spec.ts

```diff
@@ -64,6 +65,80 @@ describe('stack_utils', () => {
         })
       })
     }
+
+    it('returns the correct invocation details for a grep stack trace', () => {
+      const stack = `Error\n at itGrep (http://localhost:3000/__cypress/tests?p=cypress/support/e2e.js:444:14)\n 
+at eval (http://localhost:3000/__cypress/tests?p=cypress/e2e/spec.cy.js:14:1)\n
+at eval (http://localhost:3000/__cypress/tests?p=cypress/e2e/spec.cy.js:18:12)\n
+at eval (<anonymous>)\n
+at eval (cypress:///../driver/src/cypress/script_utils.ts:38:23)`
+
+      class GrepError {
+        get stack () {
+          return stack
+        }
+      }
+
+      stack_utils.getInvocationDetails(
+        { Error: GrepError, Cypress: { isBrowser: vi.fn(() => true) } },
+        config,
+        'test-body',
+      )
+
+      expect(source_map_utils.getSourcePosition).toHaveBeenCalledWith('http://localhost:3000/__cypress/tests?p=cypress/e2e/spec.cy.js', expect.objectContaining({
+        column: 1,
+        line: 14,
+        file: 'http://localhost:3000/__cypress/tests?p=cypress/e2e/spec.cy.js',
+      }))
+    })
+
+    it('returns the correct invocation details for a grep stack trace for a test body', () => {
+      const stack = `Error at itGrep (http://localhost:3000/__cypress/tests?p=cypress/support/e2e.js:444:14)
+    at context.notIt.only (cypress:///../driver/node_modules/mocha/lib/interfaces/bdd.js:98:46)
+    at createRunnable (cypress:///../driver/src/cypress/mocha.ts:126:31)
+    at itGrep.eval [as only] (cypress:///../driver/src/cypress/mocha.ts:187:14)
+    at Suite.eval (http://localhost:3000/__cypress/tests?p=cypress/e2e/spec.cy.js:12:6)`
+
+      class GrepError {
+        get stack () {
+          return stack
+        }
+      }
+
+      stack_utils.getInvocationDetails(
+        { Error: GrepError, Cypress: { isBrowser: vi.fn(() => true) } },
+        config,
+        'test-body',
+      )
+
+      expect(source_map_utils.getSourcePosition).toHaveBeenCalledWith('http://localhost:3000/__cypress/tests?p=cypress/e2e/spec.cy.js', expect.objectContaining({
+        column: 6,
+        line: 12,
+        file: 'http://localhost:3000/__cypress/tests?p=cypress/e2e/spec.cy.js',
+      }))
+    })
+
+    it('returns the original stack if it cannot be normalized for a test body', () => {
+      const stack = `Error at itGrep (http://localhost:3000/__cypress/tests?p=cypress/support/e2e.js:444:14)
```

Comment:
The Error should be on it's own line:

```suggestion
      const stack = `Error
    at itGrep (http://localhost:3000/__cypress/tests?p=cypress/support/e2e.js:444:14)
```

Reply:
updated